script/publish-docs

```diff
@@ -6,21 +6,15 @@ set -e
 # Store the versions
 VERSION=$(git describe --tags --match "v[0-9]*" --abbrev=0 | sed 's/^v//')
 HEAD=$(git rev-parse HEAD)
+REPO=https://x-access-token:${OCTOKITBOT_PAT}@github.com/probot/probot.github.io.git
```

Comment:
The `x-access-token:`  username prefix is for installation access tokens and GITHUB_TOKEN provided to actions only. For Personal access tokens it's not necessary. You can just do

```suggestion
https://:${OCTOKITBOT_PAT}@github.com/probot/probot.github.io.git
```

Or add the username for clarity

```suggestion
https://octokitbot:${OCTOKITBOT_PAT}@github.com/probot/probot.github.io.git
```